controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ControllerMaintenance.java

```diff
@@ -102,10 +102,10 @@ public void deconstruct() {
     /** Create one OS upgrader per cloud found in the zone registry of controller */
     private static List<OsUpgrader> osUpgraders(Controller controller) {
         return controller.zoneRegistry().zones().controllerUpgraded().zones().stream()
-                         .map(ZoneApi::getCloudName)
+                         .map(ZoneApi::getCloud)
                          .distinct()
                          .sorted()
```

Comment:
Will this work? `Cloud` is not `Comparable`?

Reply:
You're right, but I also don't see why they need to be sorted. Will remove it.